Dev/BitbucketServer/stash-pull-requests.2m.rb

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ruby
+
+require 'duration'
+require 'json'
+require 'rest-client'
+require 'pp'
+
+# rubocop:disable all
```

Comment:
totally missed that, I don't really read comments other then metadata... I've reverted my merge.  Yeah, not sure how comfortable I am with that.